src/components/elements/MapPolygonPanel/ChecklistInformation.tsx

```diff
@@ -0,0 +1,82 @@
+import Icon, { IconNames } from "@/components/extensive/Icon/Icon";
+import ModalWithMap from "@/components/extensive/Modal/ModalWithMap";
+import { useModalContext } from "@/context/modal.provider";
+
+import Button from "../Button/Button";
+import Text from "../Text/Text";
+
+const ChecklistInformation = () => {
+  const { openModal, closeModal } = useModalContext();
+  const openFormModalHandlerRequestPolygonSupport = () => {
+    openModal(
+      <ModalWithMap
+        title="Request Support"
+        onClose={closeModal}
+        content="Faja Lobi Project&nbsp;&nbsp;•&nbsp;&nbsp;Priceless Planet Coalition"
```

Comment:
Please remove all placeholder / mocked data before merging to staging.

Reply:
done

Reply:
done